### PR TITLE
Output mode = off for Adam

### PR DESCRIPTION
**Pull request by bouwew**
All changes:
- For Adam, provide mode = off, related to the regulation_mode = off, and mode = cool, for regulation_mode = cooling
- Update userdata, manual-fixture-creation, and tests related to the added modes
- A few small Typing updates
- Store updated fixtures as a result from #415 
- Manually change the adam_jip fixture, for testing in pw-beta/Core plugwise

### CODE REVIEW

**Review by CoMPaTech — COMMENTED** [review]
Nice (see dm comment on constants file)

**Review by CoMPaTech — APPROVED** [review]
Suggest we do take a quick look at how/why constants is so big as an update (to prevent future reverse of the same) before merging.

**Comment by bouwew** [comment]
Constants-file has been fixed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Changelog
 
-## Ongoing
+## v0.34.0
```
Comment (CoMPaTech) [inline CHANGELOG.md:3]: Suggest bumping to .40 as per new features

File: plugwise/__init__.py [hunk 2]
```diff
@@ -113,7 +113,7 @@ def _update_gw_devices(self) -> None:
             device.update(data)
 
             # Update for cooling
-            if device["dev_class"] in ZONE_THERMOSTATS:
+            if device["dev_class"] in ZONE_THERMOSTATS and self.smile_name != "Adam":
```
Comment (CoMPaTech) [inline plugwise/__init__.py:116]: Would it make sense to CONST Adam?

File: pyproject.toml [hunk 3]
```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name            = "plugwise"
-version         = "0.33.1"
+version         = "0.34.0a4"
```
Comment (CoMPaTech) [inline pyproject.toml:7]: Lets not forget to bump this properly

**Review by CoMPaTech — APPROVED** [review]

File: plugwise/__init__.py (hunk 2 above)
Comment (bouwew) [reply plugwise/__init__.py]: Implemented.

File: CHANGELOG.md (hunk 1 above)
Comment (bouwew) [reply CHANGELOG.md:3]: Why v0.40.0? To v0.34.0 is also a minor bump?

**Review by CoMPaTech — APPROVED** [review]

**Pull request merged**